src/main/kotlin/com/github/vol0n/utbotcppclion/server/GrpcServer.kt

```diff
@@ -1,21 +1,77 @@
 package com.github.vol0n.utbotcppclion.server
 
 import com.github.vol0n.utbotcppclion.client.GrpcStarter
+import io.grpc.Attributes
+import io.grpc.ForwardingServerCall
+import io.grpc.ForwardingServerCallListener
+import io.grpc.Metadata
 import io.grpc.Server
 import io.grpc.ServerBuilder
+import io.grpc.ServerCall
+import io.grpc.ServerCallHandler
+import io.grpc.ServerInterceptor
+import io.grpc.Status
+import kotlin.random.Random
 import kotlin.reflect.KCallable
+import kotlinx.coroutines.delay
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.flow
 import testsgen.Testgen
 import testsgen.TestsGenServiceGrpcKt
 import testsgen.Util
 import java.io.File
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
 import java.nio.file.Files
 import java.nio.file.Paths
 
+class LogInterceptor : ServerInterceptor {
+
+    companion object {
+        val LOG: Logger = LoggerFactory.getLogger(LogInterceptor::class.java)
+    }
+
+    override fun <ReqT : Any?, RespT : Any?> interceptCall(
+        call: ServerCall<ReqT, RespT>,
+        headers: io.grpc.Metadata,
+        next: ServerCallHandler<ReqT, RespT>
+    ): ServerCall.Listener<ReqT> {
+        val logServerCall = LogServerCall(call)
+        return object : ForwardingServerCallListener.SimpleForwardingServerCallListener<ReqT>(
```

Comment:
Could you please emphasize it is from grpc: use `io.grpc.ForwardingServerCallListener`